src/test/java/com/hocs/test/glue/foi/CaseCreationStageStepDefs.java

```diff
@@ -24,4 +26,16 @@ public void iEditTheCaseDetailsValue(String value) {
     public void theDetailsEnteredAreDisplayed() {
         caseCreationStage.assertCaseDetailsAreCorrect();
     }
+
+    @And("I fill the acknowledgement response date")
+    public void iFillTheAcknowledgementResponseDate() {
+        String responseDate = getTodaysDate();
+        responseDate = getDatePlusMinusNDaysAgo(-10);
+        recordCaseData.enterDateIntoDateFieldsWithHeading(responseDate, "When was the acknowledgement response to the valid request issued?");
```

Comment:
Please create a page method for interacting with page elements. We should not have this logic in the step file